src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -323,4 +331,28 @@ private void sendErrorMessageToQueue(
             messageId
         );
     }
+
+    private void logAbortedProcessingNotReadyFile(String zipFilename, String containerName) {
+        log.info(
+            "Aborted processing of zip file {} from container {} - not ready yet.",
+            zipFilename,
+            containerName
+        );
+    }
+
+    private void logAbortedProcessingNonExistingFile(String zipFilename, String containerName) {
+        log.info(
+            "Aborted processing of zip file {} from container {} - doesn't exist anymore.",
+            zipFilename,
+            containerName
+        );
+    }
+
+    private void logAbortedProcessingFilePresentInDb(String containerName, String zipFilename) {
```

Comment:
all 3 methods log almost identical info.
Perhaps you could have just one method, something like
```
logProcessingAborted(String container, String fileName, String reason)
```
just an idea.

Reply:
I was considering that myself, but decided against it, because there would be a hidden requirement for the reason to be a properly formatted string.